### PR TITLE
chore(ui): add info icon to describe file and media are not in bytes stored

### PR DESCRIPTION
**Pull request by harukatab**
## Description

- Fixes WB-NNNNN
- Fixes #NNNN

What does the PR do? Include a concise description of the PR contents.
- add an info icon that shows "This doesn't account for file and media sizes."

Screenshot 2024-10-23 at 11 26 08 AM

https://github.com/user-attachments/assets/9f5741d4-173a-4f23-89e0-2903ac73e35b

## Testing

How was this PR tested?

### CODE REVIEW

**Comment by adrnswanberg** [comment]
I would say something like "This does not take into account any files or media logged in this trace."

File: weave-js/src/components/BytesStoredInfoIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import {IconInfo} from './Icon';
+import {Tailwind} from './Tailwind';
+import {Tooltip as WeaveTooltip} from './Tooltip';
+
+export const BytesStoredInfoIcon = ({bytesStored}: {bytesStored: number}) => (
+  <Tailwind>
+    <div className="flex items-center">
+      {numeral(bytesStored).format('0.0b')}
+      <WeaveTooltip
+        content="This does not take into account any files or media logged in this trace."
```
Comment (jamie-rasmussen) [inline weave-js/src/components/BytesStoredInfoIcon.tsx:13]: This message's reference to "this trace" is confusing when used with an object version.

File: weave-js/src/components/BytesStoredInfoIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import numeral from 'numeral';
+import React from 'react';
+
+import {IconInfo} from './Icon';
+import {Tailwind} from './Tailwind';
+import {Tooltip as WeaveTooltip} from './Tooltip';
+
+export const BytesStoredInfoIcon = ({bytesStored}: {bytesStored: number}) => (
+  <Tailwind>
+    <div className="flex items-center">
```
Comment (jamie-rasmussen) [inline weave-js/src/components/BytesStoredInfoIcon.tsx:10]: Maybe add `gap-4` here? Icon feels a little crowded to me.

**Review by jamie-rasmussen — APPROVED** [review]

File: weave-js/src/components/BytesStoredInfoIcon.tsx (hunk 1 above)
Comment (harukatab) [reply weave-js/src/components/BytesStoredInfoIcon.tsx]: That's a great callout! Added another prop `weaveKind` to specify there :) 
Screenshot 2024-10-24 at 12 11 53 PM
Screenshot 2024-10-24 at 12 11 43 PM

**Pull request merged**